controller_manager:
  ros__parameters:
    update_rate: 50 # Hz

    joint_state_broadcaster:
      type: "joint_state_broadcaster/JointStateBroadcaster"

    joint1_effort_controller:
      type: "effort_controllers/JointGroupEffortController"

    joint2_effort_controller:
      type: "effort_controllers/JointGroupEffortController"

joint_state_broadcaster:
  ros__parameters:
    {}

joint1_effort_controller:
  ros__parameters:
    - joints: joint1

joint2_effort_controller:
  ros__parameters:
    - joints: joint2